src/quicktions/quicktions.pyx

```diff
@@ -607,7 +586,7 @@ cdef class Fraction:
         "Real numbers have no imaginary component."
         return 0
 
-    def conjugate(self):
+    cpdef conjugate(self):
```

Comment:
Do you really think this method is worth `cpdef`-ing?

Reply:
Probably not since there's no typing or any other cython-related code there.  It was mainly to reduce context switches between Python and Cython calls, but there's really not much going on in that method.